sys/shell/commands/shell_commands.c

```diff
@@ -268,6 +272,9 @@ const shell_command_t _shell_command_list[] = {
 #ifdef MODULE_RTT_CMD
     {"rtt", "control RTC peripheral interface",  _rtt_handler},
 #endif
+#ifdef MODULE_NANOCOAP_VFS
+    {"nget", "download a file from a CoAP server", _nanocoap_get_handler},
```

Comment:
Is `nget` a good name for this? It neither hints at the protocol nor that it is saved to the VFS -- but I gotta admit I don't have suggestions for better names.

Reply:
How about `ncget`?